### PR TITLE
Add Dockerfile and update README.md

### PR DESCRIPTION
**Pull request by saraedum**
* [x] needs #147.

### CODE REVIEW

**Comment by slel** [comment]
Suggestions re changes to `README.md`.

Line 66: is trailing whitespace on purpose?

Line 129: maybe simplify url to

  https://gitter.im/foonathan/standardese

without the "?utm..." stuff.

Line 585: change "for have supported" to
"for supporting" or "for having supported"
or "for their support of".

Line 585: capitalise "patreon" to "Patreon".

File: tool/main.cpp [hunk 1]
```diff
@@ -394,7 +394,7 @@ int main(int argc, char* argv[])
          po::value<std::string>()->default_value(""),
          "a prefix that will be added to all output files")
         ("output.format",
-         po::value<std::vector<std::string>>()->default_value(std::vector<std::string>{"html"}, "{html}"),
+         po::value<std::vector<std::string>>()->default_value(std::vector<std::string>{"commonmark"}, "{commonmark}"),
```
Comment (jzakrzewski) [inline tool/main.cpp]: This probably should be in a separate commit.

Comment (saraedum) [reply tool/main.cpp]: In principle I agree but I already add a Dockerfile and fix the README in the same commit. Sorry for that little mess.

**Comment by saraedum** [comment]
Thanks for the feedback @jzakrzewski and @slel. I'll push a docker image once I managed to convince docker hub (again @slel) that my bot user is permitted to do so…
